### PR TITLE
fix: Add chain-mon to circle ci docker release pipeline

### PR DESCRIPTION
**Pull request by roninjin10**
- Chain-mon was previously missing from docker release pipeline
- Add it

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #7836** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7836" target="_blank">Graphite</a> 
    * **PR #7837** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7837" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7837?utm_source=stack-comment).

**Comment by protolambda** [comment]
See #7811 I am currently refactoring related CI code, and with the new docker-bake file it'll be much cleaner to define docker target; you don't have to customize the build as much within the circleci config.

**Comment by roninjin10** [comment]
@protolambda I think that only applies to golang?

File: .circleci/config.yml [hunk 1]
```diff
@@ -1702,6 +1702,23 @@ workflows:
             - oplabs-gcr-release
           requires:
             - hold
+      # WIP
```
Comment (roninjin10) [inline .circleci/config.yml]:
```suggestion
```

**Review by tynes — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -1702,6 +1702,22 @@ workflows:
...
+          name: chain-mon-docker-release
+          filters:
+            tags:
+              only: /^chain-mon\/v.*/
+            branches:
+              ignore: /.*/
+          docker_file: ops/docker/Dockerfile.packages
+          docker_name: chain-mon
+          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
+          docker_target: wd-mon
```
Comment (tynes) [inline .circleci/config.yml]: should this be `wd-mon`?

Comment (tynes) [reply .circleci/config.yml]: i feel like having `n` docker images that only differ with the entrypoint is annoying to maintain

Comment (roninjin10) [reply .circleci/config.yml]: Yea this looks not good I started process of improving that here https://github.com/ethereum-optimism/optimism/pull/7840 . There should only be one image that you can pass different commands into

**Review by felipe-op — APPROVED** [review]

**Comment by protolambda** [comment]
To resolve the conflict, it should look like:
```
      - docker-release:
          name: chain-mon-docker-release
          filters:
            tags:
              only: /^chain-mon\/v.*/
            branches:
              ignore: /.*/
          docker_name: chain-mon
          docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
          context:
            - oplabs-gcr-release
          requires:
            - hold
```

And `chain-mon` is already part of the docker-bake, which defines the detail like using the `wd-mon` target.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1591,6 +1591,19 @@ workflows:
             - oplabs-gcr-release
           requires:
             - hold
+      - docker-release:
```
Comment (roninjin10) [inline .circleci/config.yml]:
```suggestion
      - docker-build:
```

**Comment by roninjin10** [comment]
Huh I didn't merge this yet?

**Comment by felipe-op** [comment]
Seems like we can close this - chain-mon is in the pipeline now

**Comment by felipe-op** [comment]
https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/32522/workflows/7b88e0c9-321d-4221-a855-73c1c7679d3d

**Pull request closed without merging**